vmdb/app/models/service/retirement_management.rb

```diff
@@ -4,7 +4,7 @@ module Service::RetirementManagement
 
   module ClassMethods
     def retirement_check
-      services = Service.all(:conditions => ["(retires_on IS NOT NULL OR retired = ?)", true])
+      services = Service.where("(retires_on IS NOT NULL OR retired = ?)", true)
```

Comment:
useless parens in the query string.  Also, the next line can just be `services.each(&:retirement_check)` :sparkles: